cli/server.go

```diff
@@ -28,30 +28,64 @@ var (
 	Version string
 )
 
-func cmdServe() *cobra.Command {
-	return &cobra.Command{
-		Use:     "serve",
-		Short:   "Serve gRPC & HTTP service",
-		Long:    heredoc.Doc(`Serve gRPC & HTTP on a port defined in PORT env var.`),
-		Aliases: []string{"server", "start"},
+func serverCmd(cfg *Config) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "server <command>",
+		Aliases: []string{"s"},
+		Short:   "Run compass server",
+		Long:    "Server management commands.",
 		Example: heredoc.Doc(`
-			$ compass serve
+			$ compass server start
+			$ compass server start -c ./config.yaml
+			$ compass server migrate
+			$ compass server migrate -c ./config.yaml
 		`),
-		Args: cobra.NoArgs,
-		Annotations: map[string]string{
-			"group:core": "true",
-		},
+	}
+
+	cmd.AddCommand(
+		serverStartCommand(cfg),
+		serverMigrateCommand(cfg),
+	)
+
+	return cmd
+}
+
+func serverStartCommand(cfg *Config) *cobra.Command {
+
```

Comment:
nit: I see couple of unnecessary new lines, for example in `serverMigrateCommand`, `func New(cliConfig *Config) *cobra.Comman`, and here